tests/Fixer/PhpUnit/PhpUnitDedicateAssertFixerTest.php

```diff
@@ -383,6 +383,14 @@ public function testAssertCountFromSizeOf($expected, $input = null)
 
     public function provideTestAssertCountCases()
     {
+//        yield 'do not fix 8' => [
```

Comment:
@SpacePossum do we want to keep this commented out?

Reply:
reverted and squashed :+1: